File: 250211/이상한 진수 2/awkward-digits-2.py
a = input()

# Write your code here!
answer = 0
num = list(a)
for i in range(len(num)):
    origin = num[i]
    if num[i] == '0':
        num[i] = '1'
    else:
        num[i] = '0'
    answer = max(answer, int("".join(num), 2))
    num[i] = origin

print(answer)

File: 260417/이상한 진수 2/awkward-digits-2.java
import java.util.*;
import java.io.*;

public class Main {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static void main(String[] args) throws IOException {
        // Please write your code here.
        String bin = br.readLine();
        String[] bin_arr = new String[bin.length()];
        int answer = 0;
        for(int i = 0; i < bin.length(); i++) {
            bin_arr[i] = String.valueOf(bin.charAt(i));
        }

        boolean flag = false;
        for(int i = 0; i < bin.length(); i++) {
            if(bin_arr[i].equals("0")) {
                bin_arr[i] = "1";
                flag = true;
                break;
            }
        }

        int temp = 0;
        for(int i = 0; i < bin.length(); i++) {
            temp += Integer.valueOf(bin_arr[i]) * Math.pow(2, bin.length()-(i+1));
        }

        if(!flag) {
            temp -= 1;
        }

        System.out.println(temp);
    }
}